test/e2e/bpmn.rendering.test.ts

```diff
@@ -98,13 +98,24 @@ describe('no BPMN elements visual regression', () => {
         },
       ],
       // ubuntu:
-      // macOS: Expected image to match or be a close match to snapshot but was 0.000987831756826818% different from snapshot
+      // macOS: Expected image to match or be a close match to snapshot but was 0.001366648725187325% different from snapshot
```

Comment:
For gateways, we have just updated the event-based gateway so I guess this is related to this.
For events, I guess the threshold hadn't been updated for a while after the latest changes we did in version `0.6.0` (https://github.com/process-analytics/bpmn-visualization-js/releases/tag/v0.6.0)